tests/table/test_snapshots.py

```diff
@@ -73,7 +73,7 @@ def test_serialize_snapshot(snapshot: Snapshot) -> None:
     )
 
 
-def test_serialize_snapshot_without_sequence_number() -> None:
+def test_serialize_snapshot_with_default_sequence_number() -> None:
```

Comment:
I feel like the initial purpose of the test is to ensure the sequence_number is not written when it is None. i.e. To test that we correctly set the pydantic model's serialization to exclude `None` field.

How about we testing this by explicitly setting the `sequence_number` as None:
```python
 snapshot = Snapshot(
...
       sequence_number=None,
...
    )
```
which reflects what we are doing now in the new `field_serializer`.

Reply:
yep good idea! done!